percy.config.yml

```diff
@@ -17,6 +17,10 @@ snapshot:
       background-color: transparent !important;
     }
 
+    #banner{
+       display: none;
+    }
+
```

Comment:
I suppose with the banner removed in `packages/karma-fixture/src/client_with_context.html` we don't need this anymore? Then we can revert all changes to this file.